### ISSUE TITLE
native-image command

### ISSUE THREAD

**Issue by damesek** [borkdude/cljtree-graalvm#1]
Hello,

In the past I run many times this script, really cool initial graalvm repo. Now, I saw.. when I would like to wrote cmd app in clojure, something strange. Worked with that command:

```
native-image --report-unsupported-elements-at-runtime \
             --initialize-at-build-time \
             -jar ./target/cljtree-graalvm-0.1.0-SNAPSHOT-standalone.jar \
             -H:Name=./target/cljtree
```

Maybe useful for others too.. (beginners) .. therefore I posted here.

Have a nice day,
Szabolcs

**Comment by borkdude** [comment]
Feel free to submit a PR if the README should be updated. Also it might be good to add the GraalVM version. Is the `--initialize-at-build-time` argument the thing you added, or are there more differences?

**Comment by damesek** [comment]
True. I'm thinking about create a graalvm - clojure docker solution (like for my yoctos projects) for this, because of this issue. I will come back on the next week.

My version was:
Java Virtual Machine: 1.8.0_212, x86_64:	"GraalVM CE 19.0.0" 
Native Image licence files (org.graalvm.native-image, version 19.0.0)

New two line: 
--report-unsupported-elements-at-runtime \
--initialize-at-build-time \ ## that solved the issue.
